patches/chrome-browser-ui-webui-favicon_source.cc.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/chrome/browser/ui/webui/favicon_source.cc b/chrome/browser/ui/webui/favicon_source.cc
+index 17eca9e53f03a55ecf4f5233b0a4dc85e73ed1b9..05cbc0f56744cdb8d5fc709e5e636a5608626b6e 100644
+--- a/chrome/browser/ui/webui/favicon_source.cc
++++ b/chrome/browser/ui/webui/favicon_source.cc
+@@ -23,9 +23,11 @@
+ #include "components/history/core/browser/top_sites.h"
+ #include "content/public/browser/browser_thread.h"
+ #include "content/public/browser/web_contents.h"
++#if !defined(OS_ANDROID)
```

Comment:
this is an upstream issue, but we should fix by adding chromium_src overrides for these 3 headers and only including the original header if `#if BUILDFLAG(ENABLE_EXTENSIONS)`

Reply:
cc: @samartnik

Reply:
this patch was removed by this commit https://github.com/brave/brave-core/commit/11aae14072622250e00240e650788d7cd8723d2e